bucket_99/rofi/manifests/plist.single

```diff
@@ -0,0 +1,39 @@
+%%DATADIR-ON%%/themes/Adapta-Nokto.rasi
+%%DATADIR-ON%%/themes/Arc-Dark.rasi
+%%DATADIR-ON%%/themes/Arc.rasi
+%%DATADIR-ON%%/themes/DarkBlue.rasi
+%%DATADIR-ON%%/themes/Indego.rasi
+%%DATADIR-ON%%/themes/Monokai.rasi
+%%DATADIR-ON%%/themes/Paper.rasi
+%%DATADIR-ON%%/themes/Pop-Dark.rasi
+%%DATADIR-ON%%/themes/android_notification.rasi
+%%DATADIR-ON%%/themes/arthur.rasi
+%%DATADIR-ON%%/themes/blue.rasi
+%%DATADIR-ON%%/themes/c64.rasi
+%%DATADIR-ON%%/themes/dmenu.rasi
+%%DATADIR-ON%%/themes/glue_pro_blue.rasi
+%%DATADIR-ON%%/themes/gruvbox-common.rasi
+%%DATADIR-ON%%/themes/gruvbox-dark-hard.rasi
+%%DATADIR-ON%%/themes/gruvbox-dark-soft.rasi
+%%DATADIR-ON%%/themes/gruvbox-dark.rasi
+%%DATADIR-ON%%/themes/gruvbox-light-hard.rasi
+%%DATADIR-ON%%/themes/gruvbox-light-soft.rasi
+%%DATADIR-ON%%/themes/gruvbox-light.rasi
+%%DATADIR-ON%%/themes/lb.rasi
+%%DATADIR-ON%%/themes/paper-float.rasi
+%%DATADIR-ON%%/themes/purple.rasi
+%%DATADIR-ON%%/themes/sidebar.rasi
+%%DATADIR-ON%%/themes/solarized.rasi
+%%DATADIR-ON%%/themes/solarized_alternate.rasi
+bin/rofi
+bin/rofi-sensible-terminal
+bin/rofi-theme-selector
+include/rofi/helper.h
+include/rofi/mode-private.h
+include/rofi/mode.h
+include/rofi/rofi-types.h
+libdata/pkgconfig/rofi.pc
+man/man1/rofi-sensible-terminal.1.gz
+man/man1/rofi-theme-selector.1.gz
+man/man1/rofi.1.gz
+man/man5/rofi-theme.5.gz
```

Comment:
This package list is full of things that need to be changed:
man/man1 => share/man/man1
%%DATADIR-ON%%, replace with full path (share/rofi probably)
libdata/pkgconfig => lib/pkgconfig

Reply:
Ok. Thanks!